frontend/src/types/crawler.ts

```diff
@@ -80,6 +80,44 @@ export type Workflow = CrawlConfig & {
   autoAddCollections: string[];
 };
 
+export type ListWorkflow = {
```

Comment:
You should be able to do `Omit<Workflow, "config">`. Check out `type Upload` for an example!

Reply:
Ah nice, thanks! :)